apitype/validator.go

```diff
@@ -0,0 +1,16 @@
+package apitype
+
+import (
+	"github.com/go-playground/validator/v10"
+)
+
+// NewValidator creates a new validator with Patch validation configured.
```

Comment:
Hmm, what does "Patch" validation mean here? Comment could probably use a little more clarification.

Reply:
For sure, thank you, will tweak that copy before merging.